infra/async_state_db/main.tf

```diff
@@ -1,3 +1,10 @@
+module "tagging" {
```

Comment:
Can we use a `locals` definition here instead of a module? It can include the `owner` and `common_tags` keys, and would not require a modification of the infra build system.

Reply:
We should be able to do that, let me remove the tagging.tf module.